.github/workflows/pull-request.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           update_packager_index: false
 
       - name: "Install GAP"
-        uses: gap-actions/setup-gap@v2
+        uses: gap-actions/setup-gap@v3
         with:
           GAP_BOOTSTRAP: 'minimal'
           GAP_PKGS_TO_CLONE: 'json'
```

Comment:
I think these two can just be removed, but next is `GAP_PKGS_TO_BUILD: 'json'` and we need to insert a step to build that package.